src/screens/Favourites/Favourites.jsx

```diff
@@ -0,0 +1,38 @@
+import Layout from "../../components/Layout/Layout";
+import TeamList from "../../components/TeamList/TeamList";
+import { useSelector, useDispatch } from "react-redux";
+import { removeAll } from "../../store/actions";
+import { NavLink } from "react-router-dom";
+
+const Favourites = () => {
+  const teams = useSelector((state) => state.favourites);
+  const isTeams = teams.length > 0;
+  const dispatch = useDispatch();
+
+  const removeAllTeam = () => {
+    dispatch(removeAll());
+  };
+
+  return (
+    <Layout>
+      <div className="teams__wrapper">
+        {isTeams ? (
+          <button
+            type="button"
+            className="btn__remove-all"
+            onClick={() => removeAllTeam()}
```

Comment:
You can use that notation if you don't pass any params to the function when you call it. It saves some resources and helps with performance.

```suggestion
            onClick={removeAllTeam}
```